app/views/users/show.html.erb

```diff
@@ -4,16 +4,18 @@
   </div>
 </div>
 
-<div class="row mb-2">
-  <div class="col-md-6 offset-md-3">
-    <%= render "users/profile_nav", user: @user %>
-  </div>
-</div>
-
-<% @user.own_photos.each do |photo| %>
-  <div class="row mb-4">
+<% if !@user.private? || current_user.leaders.include?(@user) || current_user == @user %>
+  <div class="row mb-2">
     <div class="col-md-6 offset-md-3">
-      <%= render "photos/photo", photo: photo %>
+      <%= render "users/profile_nav", user: @user %>
```

Comment:
??

Reply:
I too do not know why the diff is considering this when really it's just a difference on one line added on top